packages/rest/tests/utils/agent.ts

```diff
@@ -15,17 +15,17 @@ export async function setupAgent({
   endpoints,
   name,
   logger,
-  autoAcceptConnection,
-  autoAcceptCredential,
+  autoAcceptConnections,
+  autoAcceptCredentials,
   useLegacyDidSovPrefix,
 }: {
   port: number
   publicDidSeed: string
   endpoints: string[]
   name: string
   logger: TsLogger
-  autoAcceptConnection: boolean
-  autoAcceptCredential: AutoAcceptCredential
+  autoAcceptConnections: boolean
+  autoAcceptCredentials: AutoAcceptCredential
```

Comment:
Maybe just pass the one object that would contain the `InitConfig`. Or change some properties to be required with a custom type so that these issues will not occur.

Reply:
As this is just used for the sample and testing i have now removed most of the props to make it look cleaner.